<html>
<head>
  <title>SARS-CoV-2 Spike protein (S) gene E484K literature reference collection</title>
  <link rel="stylesheet" href="constellations.css">
</head>
<body>
<h1><a href="front.html">SARS-CoV-2</a> <a href="S.html">Spike protein (S)</a> gene <span class="constellation">S:p.E484K</span> literature reference collection</h1>
<div class="effect_section"><h2 id="ACE2_receptor_binding_affinity">ACE2 receptor binding affinity</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Reported moderate increase in affinity compared to wild-type RBD on the cell surface (Kd = 38.5nM vs 56.9nM).<br/> (<a href="https://doi.org/10.1101/2021.02.14.431117" class="lit_link">Tian et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Among the first selected variants in an in vitro evolution experiment for ACE2 binding.<br/> (<a href="https://doi.org/10.1101/2021.01.06.425392" class="lit_link">Zahradnik et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Experimentally, ACE2 binding affinity increased 0.06 fold (<a href="https://doi.org/10.1016/j.cell.2020.08.012" class="lit_link">Starr et al. (2020)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="T_cell_evasion">T cell evasion</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Analyzing responses to the E484K mutation seen in B.1.351 and P.1 variants, we noted that it did  not fall in a region predicted to bind the HLAII alleles tested (table S4). The mutation appeared  to have no substantial or differential impact on T cell responses.<br/> (<a href="https://doi.org/10.1126/science.abh1282" class="lit_link">Reynolds et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="antibody_epitope_effects">Antibody epitope effects</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Highly resistant to mAbs SARS2-01, SARS2-02, 1B07, SARS2-16, SARS2-32, SARS2-35 and 2B04 of 10 antibodies tested, and moderately high resistance to the remaining.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> For highly potent RBD-binding neutralizing antibody ADI-56443 this mutation causes ablation of neutralization efficiency.<br/> (<a href="https://doi.org/10.1101/2021.06.10.447999" class="lit_link">Haslwanter et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Multiple individuals in two related households were infected with a newly acquired E484K mutation within the B.1.311 lineage.<br/>The timing and patterns of subsequent spread were consistent  with de novo emergence of this E484K variant in the initially affected individual who had been treated with bamlanivimab monotherapy.<br/>The subsequent transmission to close contacts occurred several days after the resolution of symptoms and the end of this patient's quarantine period. (<a href="https://doi.org/10.1101/2021.10.02.21264415" class="lit_link">Sabin et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Complete loss of binding in ELISA by the variant against monoclonal antibody VH-Fc ab8 (<a href="https://doi.org/10.1101/2021.03.22.436481" class="lit_link">Sun et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Of 50 mAbs tested, major loss of neutralization observed for S2N28, S2X615, S2N12, S2X192, S2H7, S2X16, S2X58, S2H70, S2X613, S2D19, S2N22, S2D32, S2H58, S2M11, S2D106, S2X30.<br/> (<a href="https://doi.org/10.1038/s41586-021-03412-7" class="lit_link">Collier et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Monoclonal antibodies 13G9 and 58G6 maintain fairly high neutralization potency, compared to others interfacing with E484K.<br/> (<a href="https://doi.org/10.1101/2021.04.19.440481" class="lit_link">Li et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Ablates binding by class 2 mAbs such as C144 that directly interfere with ACE2 binding,  but clonal somatic mutations of memory B cells at 6.2 months (evolving humoral immune response)  show pronounced increase in binding to the variant. <br/> (<a href="https://doi.org/10.1101/2020.11.03.367391" class="lit_link">Gaebler et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Mutant screen in neutralization assay with a broad range of monoclonal antibodies shows high resistence to 4 antibodies,  and broad low level resistence against much of the rest of the panel.<br/> (<a href="https://doi.org/10.1101/2020.11.06.372037" class="lit_link">Liu et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Resistent to all seven class 2 (Spike 'up' or 'down' conformation, RBD targeting) antibodies tested, with 10-fold  or greater reduction in neutralization (plus notable reudction in two unclassfied mAbs).<br/> (<a href="https://doi.org/10.1101/2021.01.15.426911" class="lit_link">Wang et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Ablates Class 1 receptor-binding-motif targeting antibodies COV2-2050, 1B07, COVOX-384 and S2H58.<br/> (<a href="https://doi.org/10.1038/s41591-021-01294-w" class="lit_link">Chen et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Pseudotyped virus model ablates neutralization by RBD-directed mAbs 4-20, 2-4, 2-43, 2-30, 2-15, LY-Cov555, C121.<br/>Pseudotyped virus model impairs neutralization by RBD-directed mAb COV2-2196 (somewhat more than fully pseudotyped B.1.351 or live virus) (<a href="https://doi.org/10.1038/s41586-021-03398-2" class="lit_link">Wang et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Massive reduction in binding efficiency vs wild type for mAb LY-CoV555.<br/> (<a href="https://doi.org/10.1126/science.abf4830" class="lit_link">Rappazzo et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="convalescent_plasma_binding">Convalescent plasma binding</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Subtype of the B.1.526 "New York" lineage, lentivirus pseudotyped with this mutation combination in showed 3.3x reduction in  IC50 serum dilution concentration for 6 convalescent sera.<br/> (<a href="https://doi.org/10.1101/2021.03.24.436620" class="lit_link">Zhou et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> The only mutation in the B.1.351 lineage that appears to contribute to neutralization reduction (~1.7x across 10 convalescent sera from April 2020 infectees) (<a href="https://doi.org/10.1101/2021.02.05.430003" class="lit_link">Tada et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="convalescent_plasma_escape">Convalescent plasma escape</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Significant decrease in neutralization across 15 of 16 sera tested 7-17 days post symptoms onset.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> No effective neutralization in all 4 out of the 4 sera tested.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Escape mutant found after in passage in plasma pool of 26 convalescents obtained mean 1.3 months post infection.<br/> (<a href="https://doi.org/10.1101/2021.08.06.455491" class="lit_link">Schmidt et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> As measured by surface plasmon resonance, RBD with the E484K mutation alone showed a mean 19.1x decrease in binding affinity for  six batches of hyperimmune immunoglobulin (hCoV-2IG) preparations generated from SARS-CoV-2 convalescent plasma.<br/> (<a href="https://doi.org/10.1101/2021.03.19.436183" class="lit_link">Tang et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> The neutralizing activity of 15/20 convalescent sera was significantly lower against this pseudotyped virus model (<a href="https://doi.org/10.1038/s41586-021-03398-2" class="lit_link">Wang et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> This mutation occurred in 100% of sequenced virions after 12 passages and led to a 4-fold decrease in convalescent plasma neutralization activity (<a href="https://doi.org/10.1101/2020.12.28.424451" class="lit_link">Andreano et al. (2020)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="monoclonal_antibody_serial_passage_escape">Monoclonal antibody serial passage escape</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Escape variant 100% appearance in 2 passages against Regeneron monoclonal antibody REGN10989 @ 50ug/mL (99% after one passage) (<a href="https://doi.org/10.1126/science.abd0831" class="lit_link">Baum et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Strong positive selection (up to 50% of supernatant sequences) after C121 monoclonal antibody assay, decreasing in subsequent passages.<br/>Strong positive selection (up to 44% of supernatant sequences) after after one round of C144 monoclonal antibody passage, then waning on subsequent passages (<a href="https://doi.org/10.1101/2020.07.21.214759" class="lit_link">Weisblum et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> The engineered mutation cause 10-fold or more increase in the disassociation constant with many monoclonal antibodies (C144/C002/C121/C104/C110).<br/> (<a href="https://doi.org/10.1038/s41586-020-2852-1" class="lit_link">Barnes et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Mildly effective mutant against this position in the RBD for highly neutralizing COV2-2479 monoclonal antibody.<br/>Effective mutant against this position in the RBD for highly neutralizing COV2-2050 monoclonal antibody (<a href="https://doi.org/10.1016/j.chom.2020.11.007" class="lit_link">Greaney et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Escape mutation against monoclonal antibody LY-CoV555 (antibody that forms the basis for Eli Lilly's bamlanivimab) (<a href="https://doi.org/10.1101/2021.02.17.431683" class="lit_link">Starr et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Class 2 antibodies C627, C602, C671, C643, and class 2/3 antibody C603 selected for the emergence of the E484K mutation in vitro.<br/> (<a href="https://doi.org/10.1101/2021.01.15.426911" class="lit_link">Wang et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="pharmaceutical_effectiveness">Pharmaceutical effectiveness</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Bamlanivimab (LY-CoV555) lost ~16x binding against this isolated mutation.<br/>Casirivimab lost ~16x binding against this isolated mutation. (<a href="https://doi.org/10.1101/2021.04.27.440939" class="lit_link">Engelhart et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="trafficking">Trafficking</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> This variant alone shows a ~5x decrease in cell entry efficiency (RLU measurement in 293T cells) compared to D614G. <br/> (<a href="https://doi.org/10.1101/2021.05.08.443253" class="lit_link">Ferriera et al (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Lentiviral pseudotyped with this individual mutation from B.1.351 was tested on ACE2.293T cells. Luciferase activity was measured two days postinfection,  showing no change in infection rate amongst the cells.<br/> (<a href="https://doi.org/10.1101/2021.02.05.430003" class="lit_link">Tada et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="vaccine_efficacy">Vaccine efficacy</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> In this Canadian test-negative study of 324033 individuals, two doses of either mRNA vaccine (BNT162b2 (Pfizer-BioNTech) or mRNA-1273 (Moderna))  was not associated with appreciable vaccine escape due to E484K [given Dec 2020-Apr 2021 time frame of the study, this is assumed to entail both .<br/>Beta and Gamma lineages]. (<a href="https://doi.org/10.1136/bmj.n1943" class="lit_link">Chung et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="vaccine_neutralization_efficacy">Vaccine neutralization efficacy</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Neutralizing antibody titers of non-human primate sera after one or two doses of Ad26.COV2.S (Jannsen vaccine)  against the variants containing the E484K substitution in the RBD were present but reduced  (fold reduction between 3.35â€“7.78, 95% confidence interval all above twofold difference, one-sample t test). <br/> (<a href="https://doi.org/10.1084/jem.20202756" class="lit_link">Solfrosi et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> E484K pseudotyped VSV was tested for neutralization in a clonal HEK-293T ACE2 TMPRSS2 cell line optimized for highly efficient  S-mediated infection. A cohort of 12 Argentinian recipients of the Gamaleya Sputnik V Ad26 / Ad5 vaccine  showed a mean 2.8x decrease in neutralization effiacacy.<br/> (<a href="https://doi.org/10.21203/rs.3.rs-400230/v1" class="lit_link">Ikegame et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Human sera from 5 two-dose Pfizer vaccinated individuals (47-68 days post 1st-dose) neutralized this variant 3.4x less relative to reference  USA-WA1/2020 strain.<br/> 8 convalescent plasma with weak IgG ELISA titre neutralized this variant 2.4x less relative to reference  USA-WA1/2020 strain. One plasma failed to neutralize at all.<br/> 11 convalescent plasma with moderate IgG ELISA titre neutralized this variant 4.2x less relative to reference  USA-WA1/2020 strain. <br/> 11 convalescent plasma with high IgG ELISA titre neutralized this variant 2.6x less relative to reference  USA-WA1/2020 strain. <br/> (<a href="https://doi.org/10.1016/S2666-5247(21)00068-9" class="lit_link">Jangra et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Nine stored sera from Pfizer BNT162b2 vaccinees were tested against a range of spike mutation bearing PV. E484K conferred a ten-fold reduction in neutralisation by vaccine sera.<br/> (<a href="https://doi.org/10.1101/2021.05.08.443253" class="lit_link">Ferreira et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> In a cohort of 20 patients 8+ weeks after second vaccine dose of Moderna (mRNA-1273) or Pfizer-BioNTech (BNT162b2) vaccines,  ELISA tests show 10x reduced efficacy of a majority of isolated antibodies, but only a modest decrease for vaccine plasma overall.<br/> (<a href="https://doi.org/10.1101/2021.01.15.426911" class="lit_link">Wang et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="virion_structure">Virion structure</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Estimated free energy change (ddG) for this variant is -0.6 kcal/mol (i.e. destabilizing relative to wild type)  (<a href="https://doi.org/10.1101/2021.03.24.436850" class="lit_link">Spratt et al. (2021)</a>)</span></li>
</ul></div>
<div class="acknowledgement">Automatically generated using text2ui script and data from <a href="https://github.com/nodrogluap/pokay">Pokay</a></div>
</body>
</html>
