<html>
<head>
  <title>SARS-CoV-2 Spike protein (S) gene L452R literature reference collection</title>
  <link rel="stylesheet" href="constellations.css">
</head>
<body>
<h1><a href="front.html">SARS-CoV-2</a> <a href="S.html">Spike protein (S)</a> gene <span class="constellation">S:p.L452R</span> literature reference collection</h1>
<div class="effect_section"><h2 id="ACE2_receptor_binding_affinity">ACE2 receptor binding affinity</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> ~1.7-fold increase in binding affinity vs wild type.<br/> (<a href="https://doi.org/10.1101/2021.04.02.438288" class="lit_link">Motozono et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="T_cell_evasion">T cell evasion</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> This mutation causes reduced recognition by CD8 positive t-cells in people with HLA type A*24:02.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.06.006" class="lit_link">Motozono et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> L452R derivative virus did not induce IFN-gamma expression even at the highest concentration tested (10 nM) in two different A*24:02 convalescent sera  donor plasma (linear epitope NYNYLYRLF 448,456).<br/> (<a href="https://doi.org/10.1101/2021.04.02.438288" class="lit_link">Motozono et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="antibody_epitope_effects">Antibody epitope effects</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Highly resistant to mAbs SARS2-01, SARS2-02 and SARS2-32 of 10 antibodies tested.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> ~20% (ELISA significance threshold) drop in antibody binding (ELISA) by this variant against monoclonal antibody VH ab6.<br/> (<a href="https://doi.org/10.1101/2021.03.22.436481" class="lit_link">Sun et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> 10 of 14 RBD-specific mAbs that showed at least 10-fold reduced neutralization of B.1.427/B.1.429 variant pseudotype (S13I, W152C, and L452R) were also found to poorly bind to just a L452R RBD mutant, demonstrating a role for this mutation as an escape mechanism for certain RBD-targeting mAbs.<br/> (<a href="https://doi.org/10.1101/2021.03.31.437925" class="lit_link">McCallum et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Mutant screen in neutralization assay with a broad range of monoclonal antibodies shows resistence to more than one antibody.<br/> (<a href="https://doi.org/10.1101/2020.11.06.372037" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Resistent to some neutralizing antibodies: mAbs X593 and P2B-2F6 (<a href="https://doi.org/10.1016/j.cell.2020.07.012" class="lit_link">Li et al. (2020)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="convalescent_plasma_escape">Convalescent plasma escape</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> No effective neutralization in 3 out of the 4 sera tested.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Observed ~2x decrease on average in neutralization efficacy in convalescent sera of 16 health workers.<br/> (<a href="https://doi.org/10.1101/2021.04.22.441007" class="lit_link">Alenquer et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Ablation of neutralization capability of 3 of 4 convalescent sera tested, the other is significantly hindered.<br/> (<a href="https://doi.org/10.1101/2020.11.06.372037" class="lit_link">Liu et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="gene_expression_increase">Gene expression increase</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Experimentally, Spike gene expression increased 0.32 fold (<a href="https://doi.org/10.1016/j.cell.2020.08.012" class="lit_link">Starr et al. (2020)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="monoclonal_antibody_serial_passage_escape">Monoclonal antibody serial passage escape</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Escape mutation against monoclonal antibody LY-CoV555 (antibody that forms the basis for Eli Lilly's bamlanivimab) (<a href="https://doi.org/10.1101/2021.02.17.431683" class="lit_link">Starr et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Ranked effective mutant against this position in the RBD for highly neutralizing COV2-2096 (<a href="https://doi.org/10.1016/j.chom.2020.11.007" class="lit_link">Greaney et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Class 2/3 antibody C628 and class 2 antibody C643 selected for the emergence of the L452R mutation in vitro.<br/> (<a href="https://doi.org/10.1101/2021.01.15.426911" class="lit_link">Wang et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="pharmaceutical_effectiveness">Pharmaceutical effectiveness</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Inhibition of Wuhan-Hu-1 RBD point mutant pseudotypes due to 1/2 mRNA vaccines results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Inhibition of Omicron (B.1.1.529 [BA.1]) RBD point mutant pseudotypes under no vaccination results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Inhibition of Omicron (B.1.1.529 [BA.1]) RBD point mutant pseudotypes due to 1/2 mRNA vaccines results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Inhibition of Omicron (B.1.1.529 [BA.1]) RBD point mutant pseudotypes due to 3 mRNA vaccine doses results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Inhibition of Omicron BA.2 RBD point mutant pseudotypes under no vaccination results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Inhibition of Omicron BA.2 RBD point mutant pseudotypes due to 1/2 mRNA vaccines results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Inhibition of Omicron BA.2 RBD point mutant pseudotypes due to 3 mRNA vaccine doses results in high relative infection for this mutation, for individuals previously infected.<br/> (<a href="https://doi.org/10.1101/2022.08.17.504313" class="lit_link">Witte et al. (2022)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Bamlanivimab (LY-CoV555) entirely lost its neutralizing activity due to the central location of L452R in the epitopes recognized by this mAb.<br/>Regdanvimab (CT-P59), and to a smaller extent etesevimab, showed a reduction in neutralization potency. (<a href="https://doi.org/10.1101/2021.03.31.437925" class="lit_link">McCallum et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Bamlanivimab (LY-CoV555) lost ~5x binding against this isolated mutation.<br/>Cligavimab lost ~4x binding against this isolated mutation. Regdanvimab lost ~4x binding against this isolated mutation.<br/> (<a href="https://doi.org/10.1101/2021.04.27.440939" class="lit_link">Engelhart et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="trafficking">Trafficking</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> This variant alone shows a ~5x decrease in cell entry efficiency (RLU measurement in 293T cells) compared to D614G.<br/> [listed as L454R in Figure, but L452R in text, also text  suggests not statistically significant, but error bars say otherwise in Figure 4] (<a href="https://doi.org/10.1101/2021.05.08.443253" class="lit_link">Ferriera et al (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Increased stability of RBD expression in yeast, suggesting increased Spike protein stability.<br/> (<a href="https://doi.org/10.1101/2021.04.02.438288" class="lit_link">Motozono et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> We observed increased entry by pseudoviruses carrying the L452R mutation compared to D614G alone,  with a 6.7 to 22.5-fold increase in 293T cells and a 5.8 to 14.7-fold increase in human airway organoids.<br/> (<a href="https://doi.org/10.1101/2021.03.07.21252647" class="lit_link">Deng et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="vaccine_neutralization_efficacy">Vaccine neutralization efficacy</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Nine stored sera from Pfizer BNT162b2 vaccinees were tested against a range of spike mutation bearing PV. L452R conferred about a two-fold reduction in neutralisation by vaccine sera, but was not statistically significant with this sample size.<br/> (<a href="https://doi.org/10.1101/2021.05.08.443253" class="lit_link">Ferreira et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> The presence of this variant in 189 post-mRNA-vaccination COVID-19 cases was proportionally in line with lineage prevalence in Northen California during the study period, suggesting no effect of these variants on immune escape.<br/> (<a href="https://doi.org/10.1101/2021.04.14.21255431" class="lit_link">Jacobson et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="virion_structure">Virion structure</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Estimated free energy change (ddG) for this variant is -0.67 kcal/mol (i.e. destabilizing relative to wild type)  (<a href="https://doi.org/10.1101/2021.03.24.436850" class="lit_link">Spratt et al. (2021)</a>)</span></li>
</ul></div>
<div class="acknowledgement">Automatically generated using text2ui script and data from <a href="https://github.com/nodrogluap/pokay">Pokay</a></div>
</body>
</html>
