<html>
<head>
  <title>SARS-CoV-2 Spike protein (S) gene P681H literature reference collection</title>
  <link rel="stylesheet" href="constellations.css">
</head>
<body>
<h1><a href="front.html">SARS-CoV-2</a> <a href="S.html">Spike protein (S)</a> gene <span class="constellation">S:p.P681H</span> literature reference collection</h1>
<div class="effect_section"><h2 id="antibody_epitope_effects">Antibody epitope effects</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Wildtype elicits immune response, COVID-19 cohort epitope score > 99th percentile of the 497 pre-pandemic controls, mutant drops PIWAS epitope score from 7.8% to 1.2% (significantly poorer immune recognition).<br/>Together with other B1.1.7 lineage mutational changes (Spike: Y144del,N501Y, A570D Nucleoprotein: D3L, S235F) resulted in only 2 of 579 individuals (0.3% of the population) having a dramatic reduction in PIWAS antigen scores, which reflects the peak epitope signal along the entire antigen. (<a href="https://doi.org/10.1101/2021.01.06.20248960" class="lit_link">Haynes et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> This variant is adjacent to the Spike protein furin cleavage site (cleavage of S into S1 and S2 subunits is required for viral membrane fusion and subsequent entry into host cells), a site shown to be highly immunogenic.<br/> (<a href="https://doi.org/10.1101/2020.08.26.268854" class="lit_link">Johnson et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Ablates Class 3 N-terminal domain targeting antibody COV2-2489, diminishes COV2-2676.<br/> (<a href="https://doi.org/10.1038/s41591-021-01294-w" class="lit_link">Chen et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="pharmaceutical_effectiveness">Pharmaceutical effectiveness</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> This individual mutation found in the epitope from Sotrovimab causes a 0.9x reduction in neutralization efficacy using a VSV model on Vero E6 cells.<br/> (<a href="https://doi.org/10.1101/2021.03.09.434607v9" class="lit_link">Cathcart et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="trafficking">Trafficking</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> In the absence of N501Y, this Alpha lineage change in the furin clevage site did NOT show significant increase in infectivity relative to wild type,  as determined by modelling the infectivity of each Spike mutation independently in a competitive infection model of hamsters.<br/> (<a href="https://doi.org/10.1101/2021.03.08.434499" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Lentiviral pseudotyped with this individual mutation from B.1.1.7 was tested on ACE2.293T cells. Luciferase activity was measured two days postinfection,  showing NO statistically significant infection rate change amongst the cells, suggesting that furin cleavage typically used for cell entry is not affected by this change one  amino acid upstream of the RXXR recognition pattern. <br/> (<a href="https://doi.org/10.1101/2021.02.05.430003" class="lit_link">Tada et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> This mutation in the first base of the furin clevage site maintains the RXXR recognition motif, and is presumed to enhance cleavage based on the removal of a proline-directed phosphotase recognition site at S680. In a homologuous site in Infectious .<br/>Bronchitis Virus (IBV, Gammacoronaviruses), abolition of S680 phosphorylation improves furin cleavage  (and presumably cell entry).<br/> (<a href="https://www.researchgate.net/publication/348943694_The_mutation_P681H_in_the_B117_variant_of_SARS-CoV-2_probably_enhances_viral_entry_and_replication" class="lit_link">Maaroufi (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> While the introduction of P681H in the SARS-CoV-2 B.1.1.7 variant may increase spike cleavage by furin-like proteases,  this does not significantly impact viral entry or cell-cell spread. We consider that other factors are at play to account  for the increased in transmission and disease severity attributed to this variant of concern (VOC).<br/> (<a href="https://doi.org/10.1101/2021.04.06.438731" class="lit_link">Lubinski et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="virion_structure">Virion structure</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> The Ratio of S2 (processed Spike) to full length Spike is higher for this mutation,  due to a drop in the full length Spike measured, suggesting that this mutation compensates for decreased Spike production by improved proteolytic processing.<br/> (<a href="https://doi.org/10.1101/2021.02.05.430003" class="lit_link">Tada et al. (2021)</a>)</span></li>
</ul></div>
<div class="acknowledgement">Automatically generated using text2ui script and data from <a href="https://github.com/nodrogluap/pokay">Pokay</a></div>
</body>
</html>
