<html>
<head>
  <title>SARS-CoV-2 Spike protein (S) gene S477N literature reference collection</title>
  <link rel="stylesheet" href="constellations.css">
</head>
<body>
<h1><a href="front.html">SARS-CoV-2</a> <a href="S.html">Spike protein (S)</a> gene <span class="constellation">S:p.S477N</span> literature reference collection</h1>
<div class="effect_section"><h2 id="ACE2_host_variant_dependency">ACE2 host variant dependency</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> This mutation's measured interaction affinity with ACE2:p.S19P (0.03% minor allele frequency,  found almost exclusively in African/African-Americans in the GnomAD database) was significantly lower than the predicted value,  indicating that these mutations were not independent. This is consistent with the fact that the ACE2 residue S19  is adjacent to RBD residue S477 in the contact interface. <br/> (<a href="https://doi.org/10.1101/2021.05.18.444646" class="lit_link">Barton et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="ACE2_receptor_binding_affinity">ACE2 receptor binding affinity</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Experimentally, ACE2 binding affinity increased 0.06 fold (<a href="https://doi.org/10.1016/j.cell.2020.08.012" class="lit_link">Starr et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Increased affinity for ACE2: clade in Australia 2020-05-27 onward, clade in Europe 2020-08-08 onward.<br/>Modelling shows flexible region in Spike RBD with residue S477 having the greatest rotational flexibility. (<a href="https://assets.researchsquare.com/files/rs-106969/v2_stamped.pdf" class="lit_link">Singh et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Among the first selected minor variants in an in vitro evolution experiment for ACE2 binding, with increasing proportion in subsequent rounds.<br/> (<a href="https://doi.org/10.1101/2021.01.06.425392" class="lit_link">Zahradnik et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="antibody_epitope_effects">Antibody epitope effects</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Highly resistant to mAbs SARS2-01, SARS2-07, SARS2-16 and SARS2-19 of 10 antibodies tested, and moderately high resistance to the remaining.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2020)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Mutant screen in neutralization assay with a broad range of monoclonal antibodies shows some resistence across all antibodies tested.<br/> (<a href="https://doi.org/10.1101/2020.11.06.372037" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Ablates binding by class 3 mAbs such as C135 that do not directly interfere with ACE2 binding,  but clonal somatic mutations of memory B cells at 6.2 months (evolving humoral immune response)  show pronounced increase in binding to the variant. <br/> (<a href="https://doi.org/10.1101/2020.11.03.367391" class="lit_link">Gaebler et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="convalescent_plasma_escape">Convalescent plasma escape</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Significant decrease in neutralization across 11 of 16 sera tested 7-17 days post symptoms onset.<br/> (<a href="https://doi.org/10.1016/j.chom.2021.01.014" class="lit_link">Liu et al. (2021)</a>)</span></li>
<li class="lit_item"><span class="lit_summary"> Mixed bag of mild positive and negative changes in neutralization capability of all 4 convalescent sera tested.<br/>Against a wider panel of 16 convalescent plasma (no replicates), 10 show low neutralization. (<a href="https://doi.org/10.1101/2020.11.06.372037" class="lit_link">Liu et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="gene_expression_increase">Gene expression increase</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Experimentally, Spike gene expression increased 0.06 fold (<a href="https://doi.org/10.1016/j.cell.2020.08.012" class="lit_link">Starr et al. (2020)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="pharmaceutical_effectiveness">Pharmaceutical effectiveness</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> This individual mutation found in the epitope from Sotrovimab causes a 2.0x reduction in neutralization efficacy using a VSV model on Vero E6 cells.<br/> (<a href="https://doi.org/10.1101/2021.03.09.434607v9" class="lit_link">Cathcart et al. (2021)</a>)</span></li>
</ul></div>
<div class="effect_section"><h2 id="virion_structure">Virion structure</h2>
<ul class="lit_list">
<li class="lit_item"><span class="lit_summary"> Estimated free energy change (ddG) for this variant is 0.22 kcal/mol (i.e. stabilizing relative to wild type)  (<a href="https://doi.org/10.1101/2021.03.24.436850" class="lit_link">Spratt et al. (2021)</a>)</span></li>
</ul></div>
<div class="acknowledgement">Automatically generated using text2ui script and data from <a href="https://github.com/nodrogluap/pokay">Pokay</a></div>
</body>
</html>
